docs/development/mock-edc.md

```diff
@@ -0,0 +1,220 @@
+# Using the Mock-EDC for contract-based testing
```

Comment:
Maybe `Mock-Connector` would be a more meaningful name, as EDC looks too generic in this context